### PR TITLE
Feature/create board

### PR DESCRIPTION
**Pull request by edFortune**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	</head>
+	
+	<body>
+		<div id="board"></div>
+		<script type="text/javascript" src="scripts.js"></script>
+	
+	</body>
+	
+	
+</html>
```
Comment (migueliriano) [inline index.html]: Please add empty line in the end of the file.

the reason is: 

*lines not ending in a newline character aren't considered actual lines. That's why some programs have problems processing the last line of a file if it isn't newline terminated.*

File: index.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+<html>
+	<head>
+		<link rel="stylesheet" href="style.css">
+		
+	</head>
+	
+	<body>
+		<div id="board"></div>
+		<script type="text/javascript" src="scripts.js"></script>
```
Comment (migueliriano) [inline index.html]: Please can we be more specific with the name of the scripts file ? Maybe we can call this **checkers_main.js**

File: index.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+
+<html>
+	<head>
+		<link rel="stylesheet" href="style.css">
```
Comment (migueliriano) [inline index.html]: Please can we be more specific with the name of the css file ? Maybe we can call this checker_style.css

File: scripts.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+//Row 3
+for(var i = 0; i<8; i++){
+	if(i%2==0){
+		var piece = document.createElement("div");
+		piece.style.cssText = 'margin: 0%; width:50px; height:50px; background-color:#FF4500; border-radius: 50%;';
+		document.getElementById('7'+i).appendChild(piece);
+	}
+}
+
+
```
Comment (migueliriano) [inline scripts.js]: Please add empty line in the end of the file.

File: scripts.js [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+	row.style.cssText = 'display: table-row; z-index: -1;';
+
+	//This loop will add 8 blocks in each row
+	for(var j = 0; j < 8; j++ ){
+		
+			var bloc = document.createElement('div');
+
+			if(k==0){
+				if(l==0){
+					bloc.style.cssText = 'margin: 0%; width:50px; height:50px; background-color:#8B4500; display: table-cell;';
```
Comment (migueliriano) [inline scripts.js]: Can we add this style on the css file as a **Class** ? 
**i.e.:**

```css
.white-box-stile{
   margin: 0%;
   width:50px;
   height:50px;
   background-color:#8B4500;
   display: table-cell;
}
```
the that way we can only add the class name here and not repeat the style.

File: scripts.js [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+
+//Row 3
+for(var i = 0; i<8; i++){
+	if(i%2!=0){
+		var piece = document.createElement("div");
+		piece.style.cssText = 'margin: 0%; width:50px; height:50px; background-color:#5E2612; border-radius: 50%;';
+		document.getElementById('2'+i).appendChild(piece);
+	}
+}
```
Comment (migueliriano) [inline scripts.js]: we can use two`for` to populated the 3 rows the same for the **black pieces** and **white pieces**.

**i.e:**
```js
for (var row= 0; row < 3 ; row++ ){
    for (var box = 0; box < 8; box ++){
       //logic here
   }
}
```

File: scripts.js [hunk 7]
```diff
@@ -0,0 +1,111 @@
+//This k variable will determinate how the bloc has to change color differently
+//For exemle when k = 0, the white block will add first and the black will be the second in the first row
+//And when k = 1, the black will add first and the black will be the second to be added in the second row and so on...
+var k = 0;
+
+// This l variable will determinate the changing of the bloc color in a row.  
+var l = 0;
```
Comment (migueliriano) [inline scripts.js]: can you be more descriptive with the name of the variable please ?

File: scripts.js [hunk 8]
```diff
@@ -0,0 +1,111 @@
+//This k variable will determinate how the bloc has to change color differently
+//For exemle when k = 0, the white block will add first and the black will be the second in the first row
+//And when k = 1, the black will add first and the black will be the second to be added in the second row and so on...
+var k = 0;
```
Comment (migueliriano) [inline scripts.js]: can you be more descriptive with the name of the variable please ?

**Review by migueliriano — CHANGES_REQUESTED** [review]
Here some feedbacks

**Comment by edFortune** [comment]
@migueliriano  Some changes have been made to the code. Maybe you can get a look at them.

File: checkerScripts.js [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+					bloc.id = 'white-box';
+					boxChangeColor=0;
+				}
+
+				
+			}
+			row.appendChild(bloc);
+	}
+
+	if(rowChangeColor==0){rowChangeColor = 1;}else{rowChangeColor=0;}
```
Comment (migueliriano) [inline checkerScripts.js]: You can do this `shorthand`
```Javascript
var rowChangeColor = (rowChangeColor === 0) ? 1 : 0;
```

File: checkerScripts.js [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+var boardRowChildren = document.getElementById('board').children;
+
+for(var i = 0; i<8; i++){
+	for(var j = 0; j<8; j++){
+
+		//The white pieces -------------------------------------
+		if(i<3){
+			var pieceWhite = document.createElement("div");
+			if(i%2==0 && j%2==0){
+				pieceWhite.id = 'white-piece';
```
Comment (migueliriano) [inline checkerScripts.js]: Please use `CLASS` instated of `ID`.

File: checkerStyle.css [hunk 11]
```diff
@@ -0,0 +1,32 @@
+#white-box{
```
Comment (migueliriano) [inline checkerStyle.css]: Please use `CLASS` instated of `ID`
example: `.white-box`

**Review by migueliriano — CHANGES_REQUESTED** [review]
Please can you add this logic on an javascript function call  `createCheckersBoard()`?

**Comment by migueliriano** [comment]
@edFortune I left some comments :)

**Comment by migueliriano** [comment]
@edFortune  can you create a function call createCheckersBoard and put all the logic there ?

**Review by migueliriano — APPROVED** [review]

**Pull request merged**